glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -201,6 +201,21 @@ open class GleanInternalAPI internal constructor () {
         @Suppress("EXPERIMENTAL_API_USAGE")
         Dispatchers.API.flushQueuedInitialTasks()
 
+        // Check if the "dirty flag" is set. That means the product was probably
+        // force-closed. If that's the case, submit a 'baseline' ping with the
+        // reason "dirty_startup". We only do that from the second run.
+        if (!isFirstRun) {
+            @Suppress("EXPERIMENTAL_API_USAGE")
+            Dispatchers.API.launch {
+                if (LibGleanFFI.INSTANCE.glean_is_dirty_flag_set().toBoolean()) {
+                    submitPingByNameSync("baseline", "dirty_startup")
```

Comment:
Can we do `Pings.baseline.submit(Pings.baselineReasonCodes.dirtyStartup)` instead to make use of the enum?

Reply:
Oh, wait, we can't because of the sync requirement.  Carry on.